pages/challenge.js

```diff
@@ -0,0 +1,154 @@
+import axios from 'axios';
+import Section from '../components/_common_/Section/Section';
+import OutboundLink from '../components/_common_/OutboundLink/OutboundLink';
+import namesFile from '../static/operationcode_challenge/names.txt';
+import ForkButton from '../static/images/operationcode_challenge/GitHubForkButton.png';
+import ProposeButton from '../static/images/operationcode_challenge/GitHubProposeButton.png';
+import PencilIcon from '../static/images/operationcode_challenge/GitHubPencilIcon.png';
+import ExampleImage from '../static/images/operationcode_challenge/GitHubExample.png';
+import PRButton from '../static/images/operationcode_challenge/GitHubPRButton.png';
+import styles from './styles/challenge.css';
+
+const RepoLink = 'https://github.com/OperationCode/front-end/';
+const ChallengeLink = `${RepoLink}blob/master/pages/challenge.js`;
+const CompareLink = `${RepoLink}compare`;
+
+class Challenge extends React.Component {
+  constructor() {
```

Comment:
You don't need the constructor unless you're defining the initial state value based upon a prop 👍 

```suggestion
  state = {
    names: '',
  };
```